gravitee-apim-console-webui/package.json

```diff
@@ -22,8 +22,8 @@
     "@fontsource/libre-franklin": "4.4.5",
     "@gravitee/ui-analytics": "9.6.0",
     "@gravitee/ui-components": "4.1.1",
-    "@gravitee/ui-particles-angular": "9.6.0",
-    "@gravitee/ui-policy-studio-angular": "9.6.0",
+    "@gravitee/ui-particles-angular": "^9.6.0-angular-16-615de4d",
```

Comment:
We're merging with a temporary branch while we wait for cockpit validation, which will arrive shortly.